src/main/java/bio/terra/app/model/GoogleRegion.java

```diff
@@ -6,51 +6,61 @@
  * Valid regions in Google.
  */
 public enum GoogleRegion {
-        ASIA_EAST1("asia-east1"),
         ASIA_EAST2("asia-east2"),
+        ASIA_EAST1("asia-east1", GoogleRegion.ASIA_EAST2, null),
         ASIA_NORTHEAST1("asia-northeast1"),
         ASIA_NORTHEAST2("asia-northeast2"),
         ASIA_NORTHEAST3("asia-northeast3"),
         ASIA_SOUTH1("asia-south1"),
-        ASIA_SOUTHEAST1("asia-southeast1"),
         ASIA_SOUTHEAST2("asia-southeast2"),
+        ASIA_SOUTHEAST1("asia-southeast1", GoogleRegion.ASIA_SOUTHEAST2, null),
         AUSTRALIA_SOUTHEAST1("australia-southeast1"),
         EUROPE_CENTRAL2("europe-central2"),
         EUROPE_NORTH1("europe-north1"),
-        EUROPE_WEST1("europe-west1"),
         EUROPE_WEST2("europe-west2"),
         EUROPE_WEST3("europe-west3"),
-        EUROPE_WEST4("europe-west4"),
         EUROPE_WEST6("europe-west6"),
+        EUROPE_WEST1("europe-west1", GoogleRegion.EUROPE_WEST2, null),
+        EUROPE_WEST4("europe-west4", GoogleRegion.EUROPE_WEST3, null),
```

Comment:
Are the fall-back regions whatever is geographically the closest to the requested region?

Reply:
I did another check through the regions and I think so. But, I definitely wouldn't mind a second set of eyes on this list!